test/helpers/node.go

```
@@ -95,32 +96,37 @@ func (s *SSHMeta) Execute(cmd string, stdout io.Writer, stderr io.Writer) bool {
 	err := s.sshClient.RunCommand(command)
 	if err != nil {
 		log.WithError(err).Debugf("error while running command: %s", cmd)
-		return false
+		return false, err
 	}
-	return true
+	return true, nil
 }
 
 // ExecWithSudo executes the provided command using sudo privileges via SSH.
 // The stdout  and stderr of the command are written to the specified stdout and
 // stderr buffers accordingly. Returns false if execution of cmd fails.
 func (s *SSHMeta) ExecWithSudo(cmd string, stdout io.Writer, stderr io.Writer) bool {
 	command := fmt.Sprintf("sudo %s", cmd)
-	return s.Execute(command, stdout, stderr)
+	result, _ := s.Execute(command, stdout, stderr)
```

Returning only error in all `Exec*` methods would make sure that we don't ignore any errors.